client/src/features/Event/types/Event.ts

```diff
@@ -12,6 +12,26 @@ export type Event = {
   organizerName: string;
 };
 
+export type EventDetail = {
+  eventId: number;
+  title: string;
+  description: string;
+  place: string;
+  organizerName: string;
+  eventStart: string;
+  eventEnd: string;
+  registrationStart: string;
+  registrationEnd: string;
+  currentGuestCount: number;
+  maxCapacity: number;
+  questions: {
+    questionId: number;
+    questionText: string;
+    isRequired: boolean;
+    orderIndex: number;
+  }[];
```

Comment:
p2. 저라면 questions 안에 들어가는 타입을 독립시켰을 것 같아요!
아직 추가 기능이 확립되지는 않았지만, 이 타입만 골라 사용할 가능성이 0은 아니니까 분리하는 편이에요!
어떻게 생각하시나용??
```suggestion

type Questions = {
  questionId: number;
    questionText: string;
    isRequired: boolean;
    orderIndex: number;
}
  questions:  Questions
```